### PR TITLE
Many improvements to documentation

### PR DESCRIPTION
**Pull request by Gitoffthelawn**
Improved clarity, grammar, consistency, punctuation, and formatting.

Thank you for the great tool!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -772,7 +771,7 @@ Any <kbd>Ctrl</kbd> shortcut is forwarded to the device. In particular:
 This typically works as you expect.
 
 The actual behavior depends on the active application though. For example,
-_Termux_ sends SIGINT on <kbd>Ctrl</kbd>+<kbd>c</kbd> instead, and _K-9 Mail_
+`Termux` sends SIGINT on <kbd>Ctrl</kbd>+<kbd>c</kbd> instead, and `K-9 Mail`
```
Comment (rom1v) [inline README.md:774]: (I prefer italic for program names.)

File: README.md [hunk 2]
```diff
@@ -781,18 +780,18 @@ To copy, cut and paste in such cases (but only supported on Android >= 7):
  - <kbd>MOD</kbd>+<kbd>v</kbd> injects `PASTE` (after computer-to-device
    clipboard synchronization)
 
-In addition, <kbd>MOD</kbd>+<kbd>Shift</kbd>+<kbd>v</kbd> allows to inject the
-computer clipboard text as a sequence of key events. This is useful when the
-component does not accept text pasting (for example in _Termux_), but it can
-break non-ASCII content.
+In addition, <kbd>MOD</kbd>+<kbd>Shift</kbd>+<kbd>v</kbd> allows you to inject
```
Comment (rom1v) [inline README.md:783]: Is "allow injecting" also valid?

(refs https://github.com/Genymobile/scrcpy/pull/3174#discussion_r846769006)

**Review by rom1v — COMMENTED** [review]
Thank you for the improvements (I'm not native English) :+1:

Globally ok for many changes, some remarks inline.

File: README.md [hunk 3]
```diff
@@ -40,12 +40,12 @@ Its features include:
...
 The Android device requires at least API 21 (Android 5.0).
 
-Make sure you [enabled adb debugging][enable-adb] on your device(s).
+Make sure you [enable adb debugging][enable-adb] on your device(s).
 
 [enable-adb]: https://developer.android.com/studio/command-line/adb.html#Enabling
 
 On some devices, you also need to enable [an additional option][control] to
-control it using keyboard and mouse.
+control it using a keyboard and mouse.
```
[the remark this answers is not in the document]
Comment (Gitoffthelawn) [reply README.md:48]: You're welcome!

Good find, but I believe this edit is correct.  The phrase "a keyboard and a mouse" sounds awkward to me.

Neither is absolutely right or wrong (grammatically), so my preference is to not change this edit, but I'm certainly willing to to make the change you recommended.

File: README.md [hunk 4]
```diff
@@ -180,7 +180,7 @@ scrcpy --help
 
 #### Reduce size
 
-Sometimes, it is useful to mirror an Android device at a lower definition to
+Sometimes, it is useful to mirror an Android device at a lower resolution to
```
[the remark this answers is not in the document]
Comment (Gitoffthelawn) [reply README.md:183]: In English, *resolution* refers to the **number** of pixels, not their size.  I think this is what you mean here, but please let me know, as it's an important differentiation.

File: README.md [hunk 5]
```diff
@@ -372,7 +371,7 @@ scrcpy --v4l2-buffer=500    # add 500 ms buffering for v4l2 sink
 
 #### TCP/IP (wireless)
 
-_Scrcpy_ uses `adb` to communicate with the device, and `adb` can [connect] to a
+`Scrcpy` uses `adb` to communicate with the device, and `adb` can [connect] to a
```
[the remark this answers is not in the document]
Comment (Gitoffthelawn) [reply README.md:374]: Sounds good!

File: README.md (hunk 1 above)
Comment (Gitoffthelawn) [reply README.md:774]: Sounds good.  What about for `scrpy` and `adb`?

Consistency will help add clarity.  What do you think?

File: README.md (hunk 2 above)
Comment (Gitoffthelawn) [reply README.md:783]: The words *"allow injecting"* wouldn't be grammatically correct here, but *"injects"* works and is simple.

Comment (rom1v) [reply README.md:783]: Ok for "injects" :+1:

File: README.md (hunk 1 above)
Comment (rom1v) [reply README.md:774]: I would say, as a general rule:
 - _italic_ when it's the program name ("the _adb_ connection")
 - `backquotes` when it's the command ("run `adb`")

(Sometimes, the distinction is not clear.)

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (rom1v) [reply README.md:183]: OK, keep "resolution" then :+1:

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rom1v) [reply README.md:48]: OK then :+1:

File: README.md [hunk 6]
```diff
@@ -162,7 +162,7 @@ You can also [build the app manually][BUILD].
 
 ## Run
 
-Plug an Android device, and execute:
+Plug in an Android device to your computer, and execute:
```
Comment (JohnVeness) [inline README.md:165]: Hi, sorry for butting in, but I think "Plug an Android device into your computer" would be better.

File: README.md [hunk 7]
```diff
@@ -805,29 +804,29 @@ To disable automatic clipboard synchronization, use
 
 To simulate "pinch-to-zoom": <kbd>Ctrl</kbd>+_click-and-move_.
 
-More precisely, hold <kbd>Ctrl</kbd> while pressing the left-click button. Until
-the left-click button is released, all mouse movements scale and rotate the
-content (if supported by the app) relative to the center of the screen.
+More precisely, hold <kbd>Ctrl</kbd> down while pressing the left-click button.
```
Comment (JohnVeness) [inline README.md:807]: I think "hold down Ctrl" sounds better than "hold Ctrl down".

**Comment by rom1v** [comment]
I rebased, replaced some `quoted text` by _italic_, and included remarks from @JohnVeness.

 - branch [`pr3177`](https://github.com/Genymobile/scrcpy/commits/pr3177) ([commit](https://github.com/Genymobile/scrcpy/commit/pr3177))
 - [rendered](https://github.com/Genymobile/scrcpy/blob/pr3177/README.md)

Please review :wink: Is it ok for both of you?

Thank you.

**Comment by JohnVeness** [comment]
Looks good to me :+1:

**Comment by rom1v** [comment]
Merged: 3c8ebf9abd4410fd46485db43035e35b3780e8d4

**Pull request closed without merging**